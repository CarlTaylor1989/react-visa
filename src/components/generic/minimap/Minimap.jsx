import React from 'react';
import Tooltip from '../tooltip/Tooltip';

import './Minimap.scss';

const Minimap = () => (
  <Tooltip fieldName="generic.tooltips.productnetwork.minimap">
    <div className="minimap">
      <div className="inner" />
    </div>
  </Tooltip>
);

export default Minimap;
