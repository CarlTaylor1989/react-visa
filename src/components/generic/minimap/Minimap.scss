@import '../../../styles/mixins';
@import '../../../styles/variables';

$minimap-icon: '../../../styles/icons/minimap.svg';
$minimap-icon-height: 70px;
$minimap-icon-width: 110px;

.minimap {
  @include border-radius(14px);
  background-image: url($minimap-icon);
  border: 1.5px solid $colour-white;
  bottom: 112px;
  box-sizing: border-box;
  height: $minimap-icon-height;
  position: absolute;
  right: 15px;
  width: $minimap-icon-width;

  .inner {
    @include border-radius(10px);
    border: 3px solid $colour-yellow;
    bottom: 4px;
    left: 4px;
    position: absolute;
    right: 4px;
    top: 4px;
  }
}
