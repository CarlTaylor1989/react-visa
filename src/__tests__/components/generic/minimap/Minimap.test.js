/* global toJson */
import React from 'react'; // eslint-disable-line no-unused-vars
import Minimap from '../../../../components/generic/minimap/Minimap';

test('renders the Minimap component', () => {
  const wrapper = shallow(<Minimap />);
  expect(toJson(wrapper, { noKey: false })).toMatchSnapshot();
});
