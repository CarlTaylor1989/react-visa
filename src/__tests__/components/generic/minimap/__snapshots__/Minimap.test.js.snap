// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Minimap component 1`] = `
<Connect(Tooltip)
  fieldName="generic.tooltips.productnetwork.minimap"
>
  <div
    className="minimap"
  >
    <div
      className="inner"
    />
  </div>
</Connect(Tooltip)>
`;
